workshop/content/exercises/04-remediations.md

```diff
@@ -169,48 +169,48 @@ create this object in the cluster. Let's take a look at what the operator did:
 
 ```
 $ oc get machineconfigs
-NAME                                                        GENERATEDBYCONTROLLER                      IGNITIONVERSION   AGE
-00-master                                                   41d29dde06d6c72fde07b8c613743c0feb703b92   2.2.0             76m
+NAME                                               GENERATEDBYCONTROLLER                      IGNITIONVERSION   AGE
+00-master                                          522f0fa36cc7b952c6e98e120c58c66e6d795544   3.1.0             136m
 ...
-75-rhcos4-e8-worker-periodic-e8
```

Comment:
@JAORMX The previous step had us apply an individual remediation - this leads the mc to match its name instead. But later when we pause the mc pool and apply, it adds individual mcs instead of updating the single one:
```
$ oc get mc
NAME                                                          GENERATEDBYCONTROLLER                      IGNITIONVERSION   AGE
00-master                                                     522f0fa36cc7b952c6e98e120c58c66e6d795544   3.1.0             3h1m
00-worker                                                     522f0fa36cc7b952c6e98e120c58c66e6d795544   3.1.0             3h1m
01-master-container-runtime                                   522f0fa36cc7b952c6e98e120c58c66e6d795544   3.1.0             3h1m
01-master-kubelet                                             522f0fa36cc7b952c6e98e120c58c66e6d795544   3.1.0             3h1m
01-worker-container-runtime                                   522f0fa36cc7b952c6e98e120c58c66e6d795544   3.1.0             3h1m
01-worker-kubelet                                             522f0fa36cc7b952c6e98e120c58c66e6d795544   3.1.0             3h1m
75-rhcos4-e8-worker-sysctl-kernel-dmesg-restrict                                                         3.1.0             46m
75-rhcos4-e8-worker-sysctl-kernel-randomize-va-space                                                     3.1.0             25m
75-rhcos4-e8-worker-sysctl-kernel-unprivileged-bpf-disabled                                              3.1.0             25m
```

Reply:
`rhcos4-e8-worker-sysctl-kernel-kexec-load-disabled` is an example used later on, but it doesn't exist anymore for whatever reason. I used `75-rhcos4-e8-worker-sysctl-kernel-randomize-va-space` as its replacement

Reply:
That's fine! We used to have a single MC remediation which we aggregated... but that didn't work and caused bugs, so later on we switch to having an MC per-remediation.